### PR TITLE
Display events on GUI

### PR DESCRIPTION
**Pull request by cshao02**
Made significant UI changes:
1. Added an extra pane to display events on GUI.
3. Changed the layout such that the command line is at the bottom and the result display is above the command line.

Enabled execution of commands to modify eventManager such that new events are saved to JSON.

image

Perhaps more test cases need to be added though most of the changes made were UI related and testing will be time-consuming.

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -71,13 +71,13 @@ public static Event[] getSampleEvents() {
         Person[] samplePersons = getSamplePersons(); // Reuse sample persons for the event participants
 
         return new Event[] {
-            new Event("Tech Conference 2024",
+            new Event("NUS Student Life Fair 2024",
```
Comment (brianchoon) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:74]: I would kindly like to enquire the purpose of renaming this?
There is nothing wrong but I would like to know if there's a specific case you're aiming to test!

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -71,13 +71,13 @@ public static Event[] getSampleEvents() {
...
                     getPersonSet(samplePersons[0], samplePersons[1]),
                     getPersonSet(samplePersons[2]),
                     getPersonSet(samplePersons[3]),
                     getPersonSet(samplePersons[4])
             ),
-            new Event("Art Exhibition 2024",
+            new Event("NUS Open House 2024",
```
Comment (brianchoon) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:80]: A similar question as above here. I am wondering if there is a significance in this particular edit (which does not affect correctness, don't worry!)

**Review by brianchoon — APPROVED** [review]
LGTM! As mentioned by you, for the tests, we will have to add them in eventually to ensure our app is resilient amidst bugs.

**Comment by chuajunyu** [comment]
Fix #102 

Nice UI!

File: src/main/java/seedu/address/model/Model.java [hunk 3]
```diff
@@ -95,7 +95,7 @@ public interface Model {
     /**
      * Return the EventManager
      */
-    ReadOnlyEventManager getEventManager();
+    EventManager getEventManager();
```
Comment (shardhrv) [inline src/main/java/seedu/address/model/Model.java:98]: Just wanted to ask what the thought process behind this change is

**Review by shardhrv — COMMENTED** [review]
LGTM!

**Pull request merged**